etcd3/client.py

```diff
@@ -310,21 +314,26 @@ def get_prefix(self, key_prefix, sort_order=None, sort_target='key',
             metadata=self.metadata
         )
 
-        if range_response.count < 1:
-            return
-        else:
+        if return_response:
+            return range_response
+
+        def iterator():
```

Comment:
to avoid the overhead of creating a function each time these functions are called, could you move this function into utils and use it here and in the other `iterator()` locations?

Reply:
This is no longer necessary after performing the change requested in your previous comment.